demo.py

```diff
@@ -68,11 +78,26 @@ def main():
         options = here.show_exits().keys()
         directions = ", ".join(options)
 
-        print(f"It's time to move on, {player.name}, your options are: {directions}")
+        print(f"It's time to move on, {player.name}!\nYour options are: {directions}")
         direction = input("WHERE WILL YOU GO?\n> ")
 
         ok = False
         while not ok:
+            if direction.lower() == "quit" or direction.lower() == "exit":
+                print("Your vision goes blank as you collapse to the ground.")
+                sys.exit()
```

Comment:
this allows for a clean exit from the demo (instead of smashing `ctrl+c` all the time)